File: Problem1.js
/**
 * Created by sle7 on 10/1/2015.
 */

/*
    Multiples of 3 and 5
 */

var maxNum = 1000;

var sum35multiple = function(num) {
  var sum = 0;
  for (var i = 1; i < maxNum; i++) {
    if (i % 3 === 0 || i % 5 === 0) {
      sum += i;
    }
  }
  return sum;
};

console.log(sum35multiple(maxNum));

File: Problem1.py
##
# Created by sle7 on 10/1/2015.
##

##
# Multiples of 3 and 5
##

maxNum = 1000

def sum35multiple(num):
    total = 0
    for i in range(1, maxNum):
        if i % 3 == 0 or i % 5 == 0:
            total += i
    return total

print(sum35multiple(1000))
